src/displayapp/screens/Timer.cpp

```diff
@@ -6,6 +6,8 @@
 
 using namespace Pinetime::Applications::Screens;
 
+static std::chrono::seconds lastTimer;
```

Comment:
Is the global static variable needed here? Couldn't we add this variable as a member variable of the class?

Reply:
Had resolved, but latest commit re-introduces this -- if `lastTimer` is a member of the Timer app, it'll be destroyed when closing the app... Open for suggestions :)